app/models/green_lanes/exempting_additional_code_override.rb

```diff
@@ -0,0 +1,35 @@
+module GreenLanes
+  class ExemptingAdditionalCodeOverride < Sequel::Model(:green_lanes_exempting_additional_code_overrides)
+    plugin :timestamps, update_on_create: true
+    plugin :auto_validations, not_null: :presence
+
+    one_to_one :reference_additional_code, class: :AdditionalCode,
+                                           key: %i[additional_code_type_id additional_code],
+                                           primary_key: %i[additional_code_type_id additional_code] do |ds|
+      ds.with_actual(AdditionalCode)
+    end
+
+  private
+
+    def after_create
```

Comment:
Remind me the reason we are touching category assessments after any action takes place?

Reply:
Refresh cache 👍  thanks @rasikasri